src/variant.cpp

```diff
@@ -552,6 +552,22 @@ namespace {
         v->capturesToHand = true;
         return v;
     }
+    // Dragon Chess
+    // 8x8 variant invented by Miguel Illescas:
+    // https://www.edami.com/dragonchess/
+    // Like regular chess, but with an extra piece, the dragon, which moves like
+    // an archbishop (i.e. bishop+knight). The dragon can be dropped at an empty
+    // square on the back rank instead of making a normal move.
```

Comment:
The description mentions the piece can only be dropped on the back rank, but according to the rule config it can be dropped anywhere. The relevant parameters for drop regions are:
https://github.com/ianfab/Fairy-Stockfish/blob/67209f150d44b6ac5fcc01659442fcc9a4815e95/src/variant.h#L89-L90

Reply:
If you had asked me, I would have told you that the Stockfish code was too chess-specific and that you would have had better chance of success writing code from the ground up for general chess-like games rather than trying to modify Stockfish.

Fortunately, you didn't ask me. I'm amazed by how well Fairy-Stockfish works even for games very different from chess.

You are right, I messed up the drop rules. I'll fix.